src/qt/test/wallettests.cpp

```diff
@@ -113,6 +113,7 @@ void TestGUI()
         LOCK(wallet->cs_wallet);
         wallet->SetAddressBook(test.coinbaseKey.GetPubKey().GetID(), "", "receive");
         wallet->AddKeyPubKey(test.coinbaseKey, test.coinbaseKey.GetPubKey());
+        wallet->SetLastBlockProcessed(105, ::ChainActive().Tip()->GetBlockHash());
```

Comment:
missing changes in interfaces/wallet.cpp (didn't review all of 15931, moving on for now due to missing changes)